rudder-webapp/SOURCES/rudder-node-to-relay

```diff
@@ -25,19 +25,38 @@ set -e
 # documentation that, without modification of the Source Code, enables
 # supplementary functions or services in addition to those offered by
 # the Software.
-# 
+#
 # Rudder is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
-# 
+#
 # You should have received a copy of the GNU General Public License
 # along with Rudder.  If not, see <http://www.gnu.org/licenses/>.
 #
 #####################################################################################
 
 ECHO=/bin/echo
 RUDDER_OPT=/opt/rudder
+RUDDER_VAR=/var/rudder
+RUDDER_JSON="${RUDDER_VAR}/cfengine-community/inputs/rudder.json"
+
+rudder_json_value() {
+	cat "${RUDDER_JSON}" | jq -r ".$1"
+}
+
+if [ -f "${RUDDER_JSON}" ]; then
+  RUDDER_VERIFY_CERTIFICATES=$(rudder_json_value 'RUDDER_VERIFY_CERTIFICATES')
+fi
+
+if [ "${RUDDER_VERIFY_CERTIFICATES}" = "true" ]
+then
+  CERTIFICATE_OPTION=""
+else
+  CERTIFICATE_OPTION="--insecure"
+fi
+
+DOWNLOAD_COMMAND="curl --silent --show-error ${CERTIFICATE_OPTION} --location --proxy '' --globoff"
```

Comment:
use --proxy= insteal of --proxy '' to make it usable without eval

Reply:
I don't understand that part, we don't want to use the proxy, so we let the parameter empty ? If yes we can't use --noproxy ?

Reply:
--xx= is equivalent to --xx ''
but without '' which make it usable without eval
you can also use --noproxy in that case, but i guess this command was coy pasted from somewhere we didn't know the url hostname beforehand